tools/setup/emoji/build_emoji

```diff
@@ -190,6 +191,18 @@ def get_success_stamp():
     sha1_hexdigest = generate_sha1sum_emoji(ZULIP_PATH)
     return os.path.join(EMOJI_CACHE_PATH, sha1_hexdigest, 'emoji', '.success-stamp')
 
+def setup_emoji_farm(cache_path):
+    # type: (Text) -> None
+    # Copy individual emoji images from npm package.
+    src_emoji_farm = os.path.join(NODE_MODULES_PATH, 'emoji-datasource-google', 'img', 'google', '64', '*')
+    target_emoji_farm = os.path.join(cache_path, 'images-google-64')
+    run(['mkdir', '-p', target_emoji_farm])
+    run(['cp', '-RPp', src_emoji_farm, target_emoji_farm], shell=True)
+
+    # Copy zulip.png to the emoji farm.
+    zulip_image = "{}/static/assets/zulip-emoji/*".format(ZULIP_PATH)
+    run(['cp', '-RPp', zulip_image, target_emoji_farm], shell=True)
```

Comment:
I don't think we need `shell=True` for these.  You only need that if you want it to be running inside `bash`.

Reply:
I think we will need `shell=True` because we want wildcard expansion capability of the shell.